src/requests/process-method.ts

```diff
@@ -64,6 +64,23 @@ export function processMethod(method: ControllerMethod): MethodOutput {
   return {methodDef, interfaceDef, usesGlobalType, usesQueryParams};
 }
 
+/**
+ * Creates a definition of paramsSignature, which serves as input to http methods
+ * @param processedParams
+ * @param paramsType
+ */
+function getParamsSignature(processedParams: ProcessParamsOutput, paramsType: string) {
+    return (!processedParams.isInterfaceEmpty) ? `params: ${paramsType}` : '';
+}
+
+/**
+ * Creates a definition of interfaceDef, which defines interface for the http method input
+ * @param processedParams
+ */
+function getInterfaceDef(processedParams: ProcessParamsOutput) {
+    return (!processedParams.isInterfaceEmpty) ? processedParams.paramDef : '';
```

Comment:
Parentheses are not needed around 1st operand here and a couple places in the commit.

Reply:
removed from getParamsSignature and getInterfaceDef